easybuild/easyconfigs/u/ucx-py/ucx-py-setup-get_ucx_incpath.patch

```diff
@@ -0,0 +1,34 @@
+--- setup.py
```

Comment:
@zdemat please add author + short description to the top of the patch?

Reply:
Patch description and author added.